database/schemapostgres.sql

```diff
@@ -0,0 +1,17 @@
+
+DROP DATABASE IF EXISTS related_movies;
+
+CREATE DATABASE related_movies with owner = marymatthews;
+
+\connect related_movies;
+
+CREATE TABLE movies (
+  id SERIAL PRIMARY KEY,
+  title TEXT NOT NULL,
```

Comment:
Great work Mary, I was actually not sure if it was better to use VARCHAR with limits instead of TEXT but it seems TEXT is better, so, yours is good

Reply:
thanks for reviewing!